bot/src/slack.js

```diff
@@ -45,6 +59,8 @@ const updateMessage = async (event) => {
   await upsertReport({ts, message})
   await clearTags(ts)
 
+  if (await handleArchiveCommand(event.message, event.ts)) return
```

Comment:
The ts in event.message is when the message was originally sent.
I need to update the archived status with the ts of the modification.